src/MongoDB.Driver/Linq/Linq3Implementation/Misc/ConvertHelper.cs

```diff
@@ -37,6 +36,21 @@ public static Expression RemoveConvertToMongoQueryable(Expression expression)
             throw new ExpressionNotSupportedException(expression);
         }
 
+        public static Expression RemoveEnumConvert(Expression expression)
```

Comment:
Rename to `RemoveConvertToEnumUnderlyingType`

Reply:
Renamed.